pkg/controller/component/dependent.go

```diff
@@ -67,6 +131,16 @@ func (r *ReconcileComponent) addDependentResourceFull(res runtime.Object, buildF
 	}
 }
 
+func (r *ReconcileComponent) initDependentResources2(mgr manager.Manager) {
```

Comment:
This is how you can define dependent resources ? How can we then define them for different CRDs as by example the group of the k8s resources to be installed is not the same for Capability, Component Build, .... ?

Reply:
Good point. That still needs to be worked out.